File: src/Principal.java
public class Principal extends Jugador {

    public Principal(int fila, int columna) {
        super(fila, columna, "\u001B[95m A \u001B[0m");
    }

    public boolean isVivo() {
        return estaVivo;

    }

    public void setVivo(boolean vivo) {
        this.estaVivo = true;
    }

}

File: principal.js
/**
 * Clase Principal que extiende Jugador
 */
class Principal extends Jugador {
    constructor(fila, columna) {
        // Roguelike clásico: jugador como '@'
        super(fila, columna, "@");
    }

    isVivo() {
        return this.estaVivo;
    }

    setVivo(vivo) {
        this.estaVivo = true;
    }
}
